lib/src/room.dart

```diff
@@ -1535,13 +1535,16 @@ class Room {
   /// List `membershipFilter` defines with what membership do you want the
   /// participants, default set to
   /// [[Membership.join, Membership.invite, Membership.knock]]
+  /// Set [cache] to `false` if you do not want to cache the users in memory
+  /// for this session which is highly recommended for large public rooms.
   Future<List<User>> requestParticipants(
       [List<Membership> membershipFilter = const [
         Membership.join,
         Membership.invite,
         Membership.knock,
       ],
-      bool suppressWarning = false]) async {
+      bool suppressWarning = false,
+      bool cache = true]) async {
     if (!participantListComplete && partial) {
```

Comment:
Is the indentation here correct? It seems to me like this should be more indented? Wouldn't be different than before but looks wrong to me